src/main/java/org/jabref/gui/EntryTypeView.java

```diff
@@ -92,6 +95,13 @@ private void addEntriesToPane(FlowPane pane, Collection<? extends BibEntryType>
             entryButton.setUserData(entryType);
             entryButton.setOnAction(event -> setEntryTypeForReturnAndClose(Optional.of(entryType)));
             pane.getChildren().add(entryButton);
+
+            String description = getDescription(entryType);
+            if (StringUtil.isNotBlank(description)) {
+                Tooltip tooltip = new Tooltip();
```

Comment:
I would recommend refactoring the tooltip creation to the following, which prevents display issues, if some text exceeds the screen width :smiley::

```java
Screen currentScreen = Screen.getPrimary();
Rectangle2D bounds = currentScreen.getBounds();
double maxWidth = bounds.getWidth();
Tooltip tooltip = new Tooltip(description);
tooltip.setMaxWidth(maxWidth * 2 / 3);
tooltip.setWrapText(true);
entryButton.setTooltip(tooltip);
```